src/mumble/MainWindow.cpp

```diff
@@ -1674,38 +1674,50 @@ void MainWindow::on_qaUserMute_triggered() {
 
 void MainWindow::on_qaUserLocalMute_triggered() {
 	ClientUser *p = getContextMenuUser();
-	if (!p)
+	if (!p) {
 		return;
+	}
 
 	bool muted = qaUserLocalMute->isChecked();
 
 	p->setLocalMute(muted);
-	if (! p->qsHash.isEmpty())
+	if (! p->qsHash.isEmpty()) {
 		g.db->setLocalMuted(p->qsHash, muted);
+	} else {
+		logChangeNotPermanent(QString::fromLatin1("Local Mute"), p);
+	}
 }
 
 void MainWindow::on_qaUserLocalIgnore_triggered() {
 	ClientUser *p = getContextMenuUser();
-	if (!p)
+	if (!p) {
 		return;
+	}
 
 	bool ignored = qaUserLocalIgnore->isChecked();
 
 	p->setLocalIgnore(ignored);
-	if (! p->qsHash.isEmpty())
+	if (! p->qsHash.isEmpty()) {
 		g.db->setLocalIgnored(p->qsHash, ignored);
+	} else {
+		logChangeNotPermanent(QString::fromLatin1("Ignore Messages"), p);
```

Comment:
These should also get translated like so
```suggestion
		logChangeNotPermanent(QObject::tr("Ignore Messages"), p);
```
( also applies to all other instances of this function call even though I won't mark every single one)